lib/sky-pages-module-generator.js

```diff
@@ -215,6 +220,10 @@ export function SkyAppConfigFactory(windowRef: SkyAppWindowRef): any {
   return config;
 }
 
+export function skyLocaleIdFactory(localeProvider: SkyAppLocaleProvider) {
+  return localeProvider.currentLocale;
```

Comment:
Unfortunately, Angular requires that the return value of a factory be synchronous. I needed to add a synchronous option within `SkyAppHostLocaleProvider` to accommodate this requirement.